lib/elastic_record/load_from_source.rb

```diff
@@ -0,0 +1,13 @@
+module ElasticRecord
+  module LoadFromSource
+    extend ActiveSupport::Concern
+
+    included do
+      elastic_index.load_from_source = true
+    end
+
+    module ClassMethods
+      delegate :find, :find_by, :find_each, :find_in_batches, :find_ids_in_batches, :first, to: :elastic_relation
+    end
```

Comment:
I would like to see https://github.com/data-axle/content_system/blob/master/app/models/concerns/index_scope_methods.rb dealt with too.

Reply:
The whole module or the delegations?

Reply:
I gave it a try, but ran into problems with things that are ActiveRecord models but also a PercolatorModel. Pagination got screwed up.